ibis/tests/all/test_datatypes.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+import ibis.expr.datatypes as dt
+import ibis.tests.util as tu
+
+
+@pytest.mark.parametrize(('column', 'expected_dtype'), [
+    pytest.mark.xfail(
```

Comment:
I really really want the ability to skip a test based on the value of another mark. Is that currently possible with pytest?

Reply:
Could You give an example?

Reply:
We could hack a decorator which checks the markers of the test case, in similar fashion like https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option